### PR TITLE
Article about rolling upgrade issue

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: site/docs/blog/incident-2022-05.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+We implemented Nessie using this concrete global pointer mechanism, because certain decisions
+haven’t been made at that time, and we wanted to be on the “safe side” and then see what can be
+improved.
+
+## Analysis
+
+Recently we were certain that having the so-called “global state” for Iceberg tables and views is
+actually not such a great thing. So the team decided that the “global state” can go away. This was
+implemented in the Nessie PR #3866. Since that change reduced the amount of global-log-entries to
```
Comment (ajantha-bhat) [inline site/docs/blog/incident-2022-05.md:33]: #3866 suppose to be a hyper link?

File: site/docs/blog/incident-2022-05.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+## Symptom
+
+During or after a rolling upgrade from Nessie version <= 0.25.0 to >= 0.26.0, exceptions/errors like
+`org.projectnessie.versioned.ReferenceNotFoundException: Global log entry '<hex>’ not does not exist.`
+and/or `Iceberg content from reference must have global state, but has none`.
+
+## Identification of the issue and mitigation in the live system
+
+(add content - Dimitri)
```
[the remark this answers is not in the document]
Comment (snazy) [reply site/docs/blog/incident-2022-05.md:11]: No - this PR

File: site/docs/blog/incident-2022-05.md (hunk 1 above)
Comment (ajantha-bhat) [reply site/docs/blog/incident-2022-05.md:33]: >  try it - it does

I clicked on view doc (it opens markdown view), but I cannot see it as a hyperlink.

Comment (ajantha-bhat) [reply site/docs/blog/incident-2022-05.md:33]: image

Comment (dimas-b) [reply site/docs/blog/incident-2022-05.md:33]: I think PR hyperlinks work in GH only, not in the Project Nessie docs.

File: site/docs/blog/incident-2022-05.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+# Nessie issue during rolling upgrade from <=0.25.0 to >=0.26.0
+
+## Symptom
+
+During or after a rolling upgrade from Nessie version <= 0.25.0 to >= 0.26.0, exceptions/errors like
+`org.projectnessie.versioned.ReferenceNotFoundException: Global log entry '<hex>’ not does not exist.`
+and/or `Iceberg content from reference must have global state, but has none`.
```
Comment (dimas-b) [inline site/docs/blog/incident-2022-05.md:7]: I think this phase lacks a verb. Did you mean `... errors like ... may occur`?

File: site/docs/blog/index.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Nessie Blog
+
+Highlights
+
+* [Incident in May 2022](incident-2022-05.md)
```
Comment (dimas-b) [inline site/docs/blog/index.md:5]: May be rename to `Upgrading to 0.26`?

File: site/docs/blog/incident-2022-05.md (hunk 1 above)
Comment (snazy) [reply site/docs/blog/incident-2022-05.md:33]: `mkdocs serve` show the correct links.

**Comment by snazy** [comment]
+1 on @dimas-b addition

**Review by dimas-b — APPROVED** [review]
The parts I did not author LGTM :) Please someone else review my text :)

**Pull request merged**